src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml

```diff
@@ -146,144 +127,275 @@
-                                   Style="{StaticResource ItemLabelStyle}"
-                                   VerticalAlignment="Top" />
-                        <TextBlock Text="{Binding FormattedListOfPorts}"
-                                   Style="{StaticResource SelectedTextBlockStyle}"
-                                   Tag="6"
-                                   ToolTip="{Binding FormattedListOfPorts}" />
-                    </DockPanel>
-                </AdornerDecorator>
+                    <!-- Created -->
+                    <Label Grid.Row="3"
+                           Grid.Column="0"
+                           x:Uid="uidCreated"
+                           Target="{Binding ElementName=created}"
+                           Content="{x:Static local:UIResources.CreatedLabelText}"
+                           Style="{StaticResource BodyLabelStyle}" />
+                    <TextBlock Grid.Row="3"
+                               Grid.Column="1"
+                               Grid.ColumnSpan="2"
+                               x:Name="created"
+                               Text="{Binding Created}"
+                               Style="{StaticResource BodyTextBlockStyle}"
+                               ToolTip="{Binding Created}" />
+
+                    <!-- Ports -->
+                    <!-- Modified Margin on bottom -->
+                    <Label Grid.Row="4"
+                           Grid.Column="0"
+                           x:Uid="uidPorts"
+                           Target="{Binding ElementName=ports}"
+                           Content="{x:Static local:UIResources.PortsLabelText}"
+                           Margin="6,4,0,4"
+                           Style="{StaticResource BodyLabelStyle}" />
+                    <TextBlock Grid.Row="4"
+                               Grid.Column="1"
+                               Grid.ColumnSpan="2"
+                               x:Name="ports"
+                               Text="{Binding FormattedListOfPorts}"
+                               Style="{StaticResource BodyTextBlockStyle}"
+                               Margin="6,4,0,4"
+                               ToolTip="{Binding FormattedListOfPorts}" />
+                </Grid>
             </Grid>
         </DataTemplate>
     </ui:DialogWindow.Resources>
 
     <!-- Main layout grid -->
     <Grid Margin="12">
         <Grid.RowDefinitions>
-            <RowDefinition Height="Auto" /> <!--0:Connectionlabel-->
-            <RowDefinition Height="Auto" /> <!--1:Connection combo box/add new connection-->
-            <RowDefinition Height="Auto" /> <!--2:Advanced Settings-->
-            <RowDefinition Height="Auto" /> <!--3:status/refresh button-->
-            <RowDefinition Height="*" />    <!--4:ContainerList-->
-            <RowDefinition Height="Auto" /> <!--5:OK/Cancel-->
+            <RowDefinition Height="Auto" />
+            <!--0:Connection label-->
```

Comment:
><!--0:Connection label--> [](start = 12, length = 25)

These should probably be to the right or above (not below) what they are labeling